pkg/apiserver/api_installer.go

```diff
@@ -620,9 +622,10 @@ func (a *APIInstaller) registerResourceHandlers(path string, storage rest.Storag
 			addParams(route, action.Params)
 			ws.Route(route)
 		case "DELETE": // Delete a resource.
-			doc := "delete a " + kind
+			article := utilstrings.GetArticleForNoun(kind, " ")
+			doc := "delete" + article + kind
 			if hasSubresource {
-				doc = "delete " + subresource + " of a " + kind
+				doc = "delete " + subresource + " of" + article + kind
```

Comment:
@pigmej here is the actual code change. The other stuff is checked in, but not edited by hand (./hack/update-all.sh)